tests/test_structure/test_create_structure.py

```diff
@@ -0,0 +1,29 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.structure.create_structure_fixtures import (
+    office_structure_mutation_query,
+    office_structure_mutation_response,
+)
+
+
+class TestCreateOfficeStructure(BaseTestCase):
+
+    def test_structure_creation(self):
+        """
+        Testing for OfficeStructure creation
+        """
+
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            office_structure_mutation_query,
+            office_structure_mutation_response
+        )
+
+    def test_structure_creation_when_not_admin(self):
```

Comment:
Hello @daniellamarr , I suggest that we do away with this test because this test case is already handled in the code base

Reply:
Oh okay, I will do that

Reply:
But don't you think I should test when an unauthorized user is trying to access the mutation?